Cargo.lock

```diff
@@ -1825,7 +1815,7 @@ dependencies = [
  "rust-i18n-support",
  "serde",
  "serde_json",
- "serde_yml 0.0.11",
+ "serde_yml",
```

Comment:
@nc7s it looks like `rust-i18n` uses `serde_yml` unconditionally (not feature gated) so even after switching to use `toml` we still have the crate in the graph.

```
$ cargo tree -i serde_yml
serde_yml v0.0.11
├── rust-i18n-macro v3.1.2 (proc-macro)
│   └── rust-i18n v3.1.2
│       └── trippy-tui v0.13.0-dev (trippy/crates/trippy-tui)
│           └── trippy v0.13.0-dev (trippy/crates/trippy)
│               ├── hello-world v0.1.0 (trippy/examples/hello-world)
│               └── traceroute v0.1.0 (trippy/examples/traceroute)
└── rust-i18n-support v3.1.2
    ├── rust-i18n v3.1.2 (*)
    └── rust-i18n-macro v3.1.2 (proc-macro) (*)
```

Reply:
Indeed, it would be removed entirely in the next PR to replace it with i18n-embed (or other, this is the one I'm experimenting).